src/MIDebugEngine/Engine.Impl/DebuggedProcess.cs

```diff
@@ -651,7 +651,7 @@ private List<LaunchCommand> GetInitializeCommands()
                     }
 
                     if (localLaunchOptions != null &&
-                        PlatformUtilities.IsWindows() &&
+                        (PlatformUtilities.IsWindows() || (PlatformUtilities.IsOSX() && this.MICommandFactory.Mode == MIMode.Lldb)) &&
```

Comment:
I assume this problem is not limited to OS X. Maybe add a `TODO` so that we remember to remove the check for OS X if/when we add support for LLDB on Linux?